Makefile

```diff
@@ -28,7 +28,7 @@ LDFLAGS=-ldflags '-X main.VERSION=${VERSION} -X main.COMMIT=${COMMIT} -X main.BR
 all: install goinstall
 
 $(BINDATA):
-	@go get -v github.com/jteeuwen/go-bindata
+	@go get -v github.com/jteeuwen/go-bindata/...
```

Comment:
What do these ellipses do here? Wouldn't they install also all the dev dependencies of `go-bindata`? Or?

Reply:
See https://github.com/jteeuwen/go-bindata

Reply:
nice